index.html

```diff
@@ -25,70 +25,310 @@
+              <img src="images/Counter.svg" alt="counter shape"><h6 class="subhead">2022</h6>
+            </section>
+            <section class="pageone-text">
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </section>
+            <section class="buttons-section">
+              <ul >
+                <li class="js-css-html-button-ul">
+                  <button type="button" class="html-css-js-button">html</button>
+                </li>
+                <li class="js-css-html-button-ul">
+                  <button type="button" class="html-css-js-button">css</button>
+                </li>
+                <li class="js-css-html-button-ul">
+                  <button type="button" class="html-css-js-button">javascript</button>
+                </li>
+              </ul>
+            </section>
+            <section>
+              <button type="button" class="footer-button page-two-footer">See Project</button>
+            </section>
+        </section>
+      </div>
+      <div class="portfolio-container page-five-div">
+        <header class="text about-myself">
+          <h1>About</h1>
+          <h1>Myself</h1>
+        </header>
+        <section>
+          <section>
+            <p class="description">
+              Hello I’m a software developer! I can help you build a product , 
+              feature or website. Look through some of my work and experience! 
+              If you like what you see and have a project you need coded, 
+              don’t hestiate to contact me.
+             </p>
+
+          </section>
+            <h4 class="follow" >LET'S CONNECT</h4>
+            <div class="connect-icons">
+              <dl class="list container">
+                <dt class="list-item">
+                    <a href="https://twitter.com/haywayalive" target="_blank" rel="noopener noreferrer"><i class="fa fa-twitter"></i></a>
+                </dt>
+                <dd></dd>
+                
+                <dt class="list-item">
+                    <a href="https://www.linkedin.com/in/abubakar-ummar-4b6643245" target="_blank" rel="noopener noreferrer"><i class="fa fa-linkedin"></i></a>
+                </dt>
+                <dd></dd>
+                <dt class="list-item">
+                    <a href="https://medium.com/@haywayaheadshotstrategy.fx" target="_blank" rel="noopener noreferrer"><i class="fa fa-medium"></i></a>
+                </dt>
+                <dd></dd>
+                <dt class="list-item">
+                    <a href="https://github.com/Haywayaheadshot" target="_blank" rel="noopener noreferrer"><i class="fa fa-github"></i></a>
+                </dt>
+                <dd></dd>
+                <dt class="list-item">
+                    <a href="https://angel.co/u/abubakar-ummar" target="_blank" rel="noopener noreferrer"><i class="fa fa-angellist"></i></a>
+                </dt>
+                <dd></dd>
+              </dl>
```

Comment:
- [ ] Kindly make sure to wrap all social media links into the `ul` tag which will increase your work looks more semantically professional :wink: and remember to wrap each icon into an `li` tag as well.